src/app/proposals/hooks/useGetProposalsWithGraph.ts

```diff
@@ -29,60 +131,58 @@ export function useGetProposalsWithGraph() {
   const activeProposalCount = useMemo(() => {
     if (!proposalsData || !latestBlockNumber) return '0'
 
-    return proposalsData.proposals
+    return proposalsData
       .filter(
-        (proposal: ProposalGraphQLResponse) =>
+        (proposal: ProposalApiResponse) =>
           handleProposalState(proposal, latestBlockNumber) === ProposalState.Active,
       )
       .length.toString()
   }, [proposalsData, latestBlockNumber])
 
   const totalProposalCount = useMemo(() => {
     if (!proposalsData) return '0'
-    return proposalsData.counters.find(e => e.id === 'proposals')?.count || '0'
+    return proposalsData.length.toString()
   }, [proposalsData])
 
   const activeProposals: Proposal[] = []
   const inactiveProposals: Proposal[] = []
 
-  const proposalResponse = proposalsData?.proposals.map(proposal => {
-    const againstVotes = Big(proposal.votesAgainst).div(Big('1e18')).round()
-    const forVotes = Big(proposal.votesFor).div(Big('1e18')).round()
-    const abstainVotes = Big(proposal.votesAbstains).div(Big('1e18')).round()
-    const deadlineBlock = Big(proposal.voteEnd)
-    const creationBlock = Number(proposal.createdAtBlock)
-    const eventArgs = getProposalEventArguments({
-      args: {
-        description: proposal.description,
-        proposalId: BigInt(proposal.proposalId),
-        proposer: proposal.proposer.id as Address,
-        targets: proposal.targets,
-        values: proposal.values.map(value => (value ? BigInt(value) : 0n)),
-        calldatas: proposal.calldatas,
-        voteStart: BigInt(proposal.voteStart),
-        voteEnd: BigInt(proposal.voteEnd),
-      },
-      timeStamp: proposal.createdAt,
-      blockNumber: proposal.createdAtBlock,
-    })
-    const { calldatasParsed } = eventArgs
-    const category = getProposalCategoryFromParsedData(calldatasParsed, proposal.description)
+  const proposalResponse = proposalsData?.map((proposal: ProposalApiResponse) => {
```

Comment:
I don't think this name reflects what this variable is.

I think this is more of a proposalTransformedResponse as it's going through data transformation.